spec/support/solidus_stripe/checkout_test_helper.rb

```diff
@@ -32,8 +32,16 @@ def payment_method
     @payment_method ||= SolidusStripe::PaymentMethod.first!
   end
 
+  def current_order
+    Spree::Order.last
+  end
+
+  def last_stripe_payment
+    Spree::Payment.find_by(source_type: "SolidusStripe::PaymentSource")
```

Comment:
This should actively select the last payment, probably better to fetch it from `current_order.payments`